spacemesh/v1/admin_types.proto

```diff
@@ -14,3 +16,91 @@ message RecoverRequest {
   string uri = 1;
   uint32 restore_layer = 2;
 }
+
+message Event {
+  // time when event occured.
+  google.protobuf.Timestamp timestamp = 1;
+  // the reason of the failure may be complex to pinpoint.
+  // for the first version we want to highlight that failure occured and defer user to logs. 
+  bool failure = 2;
+  // what role this event plays in the protocol. 
+  // might be long, should not be displayed by default, or displayed in small text.
+  string help = 3;
```

Comment:
this should go as docstring for events below

Reply:
decided to keep it to simplify integration, in future @brusherru may use better messages in smapp